core/META-INF/MANIFEST.MF

```diff
@@ -3,9 +3,9 @@ Bundle-ManifestVersion: 2
 Bundle-Name: LibreOffice eclipse integration - core
 Bundle-SymbolicName: org.libreoffice.ide.eclipse.core;singleton:=true
 Bundle-Vendor: The Document Foundation
-Bundle-Version: 3.0.0
+Bundle-Version: 3.0.1
```

Comment:
Please no version change in this patch. Version will be updated when a release is done.